docs/getting-started.md

```diff
@@ -75,12 +75,85 @@ KONNECTD_DEBUG_ZPAGES
 KONNECTD_HTTP_ADDR
 : Address to bind http server, defaults to `0.0.0.0:9130`
 
-KONNECTD_HTTP_NAMESPACE
-: The http namespace
-
 KONNECTD_HTTP_ROOT
 : Root path of http server, defaults to `/`
 
+KONNECTD_HTTP_NAMESPACE
+: Set the base namespace for service discovery, defaults to `com.owncloud.web`
+
+KONNECTD_IDENTITY_MANAGER
+: Identity manager (one of ldap,kc,cookie,dummy), defaults to `ldap`
```

Comment:
are we keeping the `dummy` thing?

Reply:
asking because it makes for a strange interface: https://github.com/owncloud/ocis-konnectd/blob/2dcddd2286637036302a3dcc0661082bb01e57ac/pkg/service/v0/service.go#L26-L29